test/pkg/environment/common/environment.go

```diff
@@ -92,6 +95,15 @@ func NewEnvironment(t *testing.T) *Environment {
 	// Get the output dir if it's set
 	outputDir, _ := os.LookupEnv("OUTPUT_DIR")
 
+	instanceTypes, err := kwok.ConstructInstanceTypes(ctx)
+	if err != nil {
+		log.Default().Printf("failed constructing instance types: %v", err)
```

Comment:
Shouldn't this also be a fatal error?
```suggestion
		log.Default().Printf("failed constructing instance types, %w", err)
```